README.md

```diff
@@ -3,765 +3,222 @@
+- [Tools](##Tools)
+- [License](##License)
 
-1. Copy `.env-api.sample` as `.env-api` and `.env-issuer.sample` as `.env-issuer`. Please see the [configuration](#configuration) section for more details.
-1. Run `make up`. This launches 3 containers with Postgres, Redis and Vault. Ignore the warnings about variables, since those are set up in the next step.
-1. **If you are on an Apple Silicon chip (e.g. M1/M2), run `make run-arm`**. Otherwise, run `make run`. This starts up the issuer API, whose frontend can be accessed via the browser (default <http://localhost:3001>).
-1. [Add](#import-wallet-private-key-to-vault) your Ethereum private key to the Vault.
-2. [Setup vault](#setup-vault).
-3. [Create](#create-issuer-did) your issuer DID.
-4. _(Optional)_ To run the UI with its own API, first copy `.env-ui.sample` as `.env-ui`. Please see the [configuration](#development-ui) section for more details.
-5. _(Optional)_ Run `make run-ui` (or `make run-ui-arm` on Apple Silicon) to have the Web UI available on <http://localhost:8088> (in production mode). Its HTTP auth credentials are set in `.env-ui`. The UI API also has a frontend for API documentation (default <http://localhost:3002>).
+## Quick Start Installation
+> [!NOTE]
+> The provided installation guide is **non-production** ready. For production deployments please refer to  [Standalone Mode Guide](https://devs.polygonid.com/docs/issuer/setup-issuer-core/).
+>
+> There is no compatibility with Windows environments at this time.
 
-#### Docker Guide Requirements
+### Prerequisites
 
 - Unix-based operating system (e.g. Debian, Arch, Mac OS)
 - [Docker Engine](https://docs.docker.com/engine/) `1.27+`
 - Makefile toolchain `GNU Make 3.81`
+- Publicly accessible URL - The issuer node API must be publicly reachable. Please make sure you properly configure your proxy or use a tool like [Localtunnel](https://theboroer.github.io/localtunnel-www/) for testing purposes.
+- Polygon Mumbai or Main RPC - You can get one in any of the providers of this list
+    - [Chainstack](https://chainstack.com/)
+    - [Ankr](https://ankr.com/)
+    - [QuickNode](https://quicknode.com/)
+    - [Alchemy](https://www.alchemy.com/)
+    - [Infura](https://www.infura.io/)
 
-> **NOTE:** There is no compatibility with Windows environments at this time.
+### Issuer Node Installation
 
-To help expedite a lot of the Docker commands, many have been abstracted using `make` commands. Included in the following sections are the equivalent Docker commands that show what is being run.
+In this section we will cover the installation of the Issuer Node API.
 
-#### Create Docker Configuration Files
+> [!NOTE]
+> This Quick Installation Guide is prepared for Polygon Mumbai (Testnet) both for the state contract and issuer dids. If you want to deploy the node with Polygon Main configuration, please visit our [advanced Issuer Node configuration guide](https://devs.polygonid.com/docs/issuer/issuer-configuration/)).
```

Comment:
This link is broken.

Reply:
it's broken until the official docs gets deployed

Reply:
I'll leave this open until it's deployed and tested so we don't miss it.